frontend/src/app/main/data/workspace/groups.cljs

```diff
@@ -188,7 +188,9 @@
             (->> ids
                  (cfh/clean-loops objects)
                  (remove #(ctn/has-any-copy-parent? objects (get objects %)))
-                 (shapes-for-grouping objects))
+                 (shapes-for-grouping objects)
+                 (remove #(ctk/is-variant? %)))
```

Comment:
Don't need to create an anonymous function, just pass `ctk/is-variant?` to remove.